### ISSUE TITLE
Codeblocks are shrinked

### ISSUE THREAD

**Issue by Kermalis** [DevEvil99/DarkPlus-Discord-Theme#6]
I love this theme, but the only bad part about it is the codeblocks are small. I thought it was a plugin for months, but I just realized it was this theme... So here I am

...

Here is no theme:
![image](https://user-images.githubusercontent.com/29823718/178083155-9e84c8de-5048-4bb3-95b5-549bf0c9099c.png)

Here is Dark+:
![image](https://user-images.githubusercontent.com/29823718/178083169-0aebeb80-f0ad-4cd1-9d51-53504b8c930c.png)

This is especially annoying for putting actual code. It makes it very hard to follow for me

I tried to find a good example of it being hard to read, but I can't find one at the moment. But this screenshot shows how annoying it can be even if it's not hard to read:

No theme (I can read it like a text editor):
![image](https://user-images.githubusercontent.com/29823718/178083304-d3f7456f-9025-44cc-98cc-0e9c165060e3.png)
Dark+ (Looks like a phone's text editor):
![image](https://user-images.githubusercontent.com/29823718/178083284-4da0786a-7cc7-4ef7-b9c4-d3a9efd6d629.png)

If you fix this, this will basically be the perfect theme 🥲

**Comment by DevEvil99** [comment]
I will look into it, thank you for your feedback!

**Comment by DevEvil99** [comment]
fixed

**Comment by Kermalis** [comment]
Is there a way to refresh the cache, or does it apply automatically?

It looks different now (the inner part stretches but the size didn't fix):
![image](https://user-images.githubusercontent.com/29823718/178126707-20b0e0cd-7aef-4835-a58a-bf716e1c4956.png)

**Comment by DevEvil99** [comment]
im on vacation rn, i will fix it soon!

**Comment by Kermalis** [comment]
Bump in case you forgot

**Comment by DevEvil99** [comment]
fixed

**Comment by Kermalis** [comment]
Thanks :)
